app/schema.py

```diff
@@ -12,13 +12,34 @@ class Meta:
         model = User
 
 
+class FriendRequestType(DjangoObjectType):
+    class Meta:
+        model = FriendRequest
+
+
+class FriendType(DjangoObjectType):
+    class Meta:
+        model = Friend
+
+
 class Query(ObjectType):
     user = graphene.Field(UserType, contact=graphene.String())
+    friend_request = graphene.Field(FriendRequestType, contact_receiver=graphene.String())
```

Comment:
Do one thing, split this into two queries, one for received and one for sent. Also to send a list of objects, instead of `graphene.Field`, use `graphene.List`